dxr/static/js/code-highlighter.js

```diff
@@ -7,13 +7,13 @@
  * 1) Multi-select highlight lines with shift key and update window.location.hash
  * 2) Multi-select highlight lines with command/control key and update window.location.hash
  * 3) Highlight lines when page loads, if window.location.hash exists
- * In addition, we update the permalink link to keep it synchronized with window.location.
+ * In addition, we update links to keep them synchronized with window.location.
  */
 
 $(function () {
     'use strict';
     var container = $('#line-numbers'),
-        permalink = $('.permalink'), // whenever we update window.location, update this href too
+        syncLinks = $('.sync-link'), // whenever we update window.location, update these too
```

Comment:
If we go down the token replacement road, we'll want to look up how efficient a[data-template] selectors are. Maybe we don't need a marker class.